src/plugins/controls/public/options_list/options_list_reducers.ts

```diff
@@ -29,15 +32,13 @@ export const optionsListReducers = {
       state.explicitInput.selectedOptions = newSelections;
     }
   },
-  deselectOptions: (
-    state: WritableDraft<OptionsListReduxState>,
-    action: PayloadAction<string[]>
-  ) => {
+  deselectOptions: (state: WritableDraft<OptionsListReduxState>, action: PayloadAction<string>) => {
```

Comment:
Why have you changed the payload action type here from `string[]` to `string`?

Reply:
Woops! That was probably leftover from playing around with storing the document count as part of the selections array 🙇  It didn't break anything cause it looks like we never actually use `deselectOptions`, we only ever use `deselectOption`.... Do you know why this reducer even exists, now that you mention it? 👀 

Will change this back to `string[]` though, regardless

Reply:
Hurm. If it's never used we should probably delete it. It might be a holdover from some earlier development.

Reply:
Good call! Will do :+1: